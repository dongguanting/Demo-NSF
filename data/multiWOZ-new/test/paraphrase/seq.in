it  'd like to find a train whose departure in on monday and arrives by 19:00
i want to leave from peterborough
i just need one
departing from stansted airport after 11:45
i 'd like to depart after 19:45 from stevenage
do you know when the train leaves on tuesday
i want  go from cabridge to london liverpool street
anytime after 20:15 is good
are there any trains going to cambridge
iam leaving for broxbourne on tuesday
as long as i arrive by 1645
yes i would like one ticket
i want to go to cambridge by train i have to get there at approximately 11:30
leaving from ely
i 'm going to leicester from cambridge on saturday
i mean to say evening 20:45
i need to reach by 19:30 if it 's not too much trouble
may i know what trains depart from london kings cross
on saturday leaving london kings cross and going to cambridge
the train should be heading towards cambridge and arrive there by 14:00
please book for 5 of us
i am leaving on monday after 1630
11:15 arrival
i 'd like a train that will arrive on wednesday at 12:30
i am leaving birmingham new street and heading to cambridge
yes 12:30 is the time i have to be in cambridge please
i 'll be leaving saturday
i need to take a train after 17:30 leaving cambridge
traveling to stansted airport on saturday
i want to travel by train on monday from cambridge
i want to reserve three train seats please
do you have a train to liverpool street london leaving after 515pm
i am hoping to leave on friday
i 'm searching for the train that leaves from cambridge on thursday
ok so you need one for 145pm right
my departure station is cambridge
yes i am hoping to go to birmingham new street
i 'm going on monday
just a single ticket please
i am departing from liverpool street station in london
i would like to leave on wednesday
can i please find a train that will be leaving cambridge on monday
i will be leaving kings cross after 17:15
i have to go out from cambridge by train on sunday
i am looking for the train arriving in stevenage at 11:45
find a train departing thu for stansted airport
is there a train that will be leaving from norwich
i need to travel to cambridge this tuesday
im leaving norwich on wednesday
i want to do the same for 4 more people
i would like to travel from cambridge to stansted please
at 13:45 i want to leave
today i need one ticket
the train i need is the one from cambridge to bishops stortford
book one ticket only
i am going to london king cross and i have to arrive before 16:45
help me please to find train to cambridge
i 'm looking for a train that would leave from london liverpool after 20:00 on friday
may i know what trains depart from cambridge
can you tell me about the trains on sunday from cambridge
so to verify- it will arrive in stevenage by 9:00
im looking for a train leaving cambridge
on sunday i will be going to liverpool street in london
i want to take a train to cambridge
does the train going to cambridge depart from broxbourne station
i am planning to leave on sunday
i want to leave after 13:45
i would like information for a train heading from cambridge to broxbourne
i need to catch the first train after 19:45 may i have its id and price
i need to go to stansted airport by train from cambridge station
i want a train to cambridge
i will be leaving for cambridge around 800
that works can i get 6 tickets
please help me to find cambridge-norwich train
i am going to leave on sunday
i need to get there by 12:30
i need only one ticket please
yes i want to travel on sunday
i think some evening perhaps after 17:15
please book for 2 people
i want to depart from cambridge to birmingham new street on monday by 12:45
which train leaves on thursday and arrived by 9:15
iam from cambrdrige  where am i now
i need a train going to birmingham new street leaving thursday
i am leaving from cambridge and i need to arrive by 17:15
from cambridge to london liver pool
i would like arrive 15:45 on tuesday
i would like a saturday train out of cambridge
i 'd like to go to kings lynn
i am in need of train transportation between stansted and cambridge
is there a train available line london liverpool street after 1930
please tell me if there are any trains to ely
for thursday
i 've got to get there no later than 14:15
i am travelling to cambridge from broxbourne
i would like to leave on saturday
i would like to leave by 08:30
okay 6 reservations please
i need to go to norwich by train this monday
i need a train that leaves after 17:00 and goes to cambridge
a train has to be there by 13:00
i need seats for 4
on wednesday i am travelling to ely & i should be there around 17:15
can i get a train from cambridge after 0915
id like to leave kings lynn on monday and arrive at cambridge at 1445
tell me departure time of the train arrives at 14:45
i am searching for a train to london kings cross that departs after 08:15
i 'm looking for train from cambridge after 20:45
oh i mean to london somewhere at liverpool street
yes a thursday departure would be best
gonna take the train after 16:00 on wendesday
i want to go to kings lynn
the arrival time should by 9:45
leaving on sunday would be ideal
i mean birmingham new street as point of destination on wednesday
i am looking for a train to cambridge
the train should arrive by 21:30 at stansted airport
i need to get a train from cambridge to stansted airport on tuesday
the train that goes from cabridge to the stansted airport on tuesday arriving the closest to 21:30
i would like a train which leave early on wednesday
i 'd like to book that for 8 people and i also need a reference number
i am attempting to locate my train
i would like one departuring from cambridge and going to kings cross
user : i am looking for a train agent : i 'd be happy to help you with that can you tell me your departure location and where you are headed to user : i am looking to go to london kings cross and depart from cambridge agent : there are 70 trains making that trip do you have a day or time
i want to leave from broxbourne after 12:30
i am looking for a train that leave after 15:15 to got to broxbourne
i am looking for a train on sunday afternoon
i need help getting train tickets for tuesday
i am heading to cambridge from stevenage around 11 in the morning
on wednesday i will need a train
i need a train to liverpool street that leaves after 10:00
i 'm leaving from cambridge friday
which train is departing from broxbourne
i want to go to cambridge
i need to arrive by 1715
i 'm departing on wednesday
does the tr0788 arrive in cambridge by 17:15 latest
i am looking for a train to cambridge
i am planning to leave ely this friday
if we can leave after 09:45
i depart on saturday morning at 8:45
i need to leave cambridge on thursday can i have a ticket
i need to have a departure after 20:15
i need one seat
please provide me with routes that leave from norwich and arrive at cambridge
at 14:15 i want to be in cambridge
there are two passenger
can you tell me about the trains leaving on friday to cambridge
could you help me find a train from cambridge
can you tell me how to get to stansted airport please
i am interested in traveling thursday please
i have to leave later than 16:15
i am looking to book a ticket for 2 people for the train on tuesday arriving at 10:30 am in broxbourne and departing cambridge
i am travelling on monday and will arrive at 14:15
on monday i need to travel after 12:00
i would like to arrive in leicester
i need to be in cambridge monday
only one please
i want a ticket for the train leaving from cambridge and arrives in peterborough by 21:45
i need a train wednesday from liverpool station
i need to be there by 14:45
i would like to depart on wednesday
sure i would like to reserve it for five people is that possible
i need to identify a train leaving from cambridge to peterborough
my destination is kings lynn
i am going to be coming from cambridge
the train should arrive at peterborough and depart after 10:15
i 'm in cambridge need to get to ely
i need a train that will arrive in peterborough by 18:30
i would like to depart from cambridge on sunday
i am searching for a train arriving at cambridge
i 'm trying to find a train goind to cambridge
the user says he will be leaving from brookshite
i need to be taken to cambridge from london liverpool street
user ask the agent for train booking from to reach the london liverpool street to cambridge
i need one ticket please
could you tell me the time when the train will arrive in london liverpool street
no i have to depart on saturday
i 'm looking for a train from cambridge to ely for this thursday
im leaving at 5:45 after would be awesome
i need a train from cambridge to leicester
i 'm leaving on monday
i 'm looking for a train to peterborough
on the next train after 11:15 could you book for 6 people
i 'd like to leave on sunday after 9:00
i 'm heading to cambridge
i will be leaving from the london liverpool street station
i want to leave london after 9pm
i would like to leave cambridge at a time that will allow me to arrive in king lynn at 8:00
iâ€™m looking for a train that is going to cambridge
looking to get there by 13:00 sunday
i 'm need a train heading to broxbourne please
ill need a train going to london kings cross for wednesday
i need it to get here by 18:00
see if you can locate a train from peterborough to cambridge that arrives by 8:45
i 'd like to travel on saturday from londo liverpool street
are there any trains from cambridge to ely
yes could you book for 2 more people please and get me a reference number as well
i would like to book a train trip
i 'm at bishops stortford looking for cambridge
wednesday looking to depart after 08:15
the user wants to find a train that will leave on sunday
wednesday is the travel date
i need to arrive by 18:15 to make an appointment
i`ll be going from cambridge to broxbourne
i 'm seeking out for a train to take 5me to cambridge
i like to leave from stevenage and reach cambridge by 9:15 on thursday
i 'm departin leicester and must leave no later than 20:30
need a train out of king lynn on tuesday
i would like to depart tuesday
train leaving tuesday after 9:30 kings lynn to cambridge
for saturday i need a train going to liverpool street london
looking for train leaving cambridge after 15:30
going to cambridge
im going from cambridge to peterborough
yes book a ticket for one and give me the reference number
which train is available after 1145
i will be traveling from stansted airport to cambridge on friday
the offer told to the user would be good for him and even more if he could get a train for 7 people
the train should leave after 14:15 to birmingham new street
i 'm searching for a train arriving at cambridge close to 10:45
i need a train from cambridge to norwich
do you know if there are any thursday trains departing from cambridge
my destination is broxbourne
i wan the one that will arrive at 3pm
i will have 6 tickets
i am looking for a train to peterborough from cambridge this sunday
i am looking to depart after 18:15
iâ€™m departing norwich
i will be departing after 14:30
i would like to book a train with departure time after 9:15 from cambridge
i am searching for a train that would take me to cambridge from london liverpool street  on thursday
i need to take a train to cambridge
yea i was standing and waiting for a trainand its was coming from kings lynn
thursday is the date i 'm looking to arrive
i 'm looking to depart after 10:00
i would like to find a train
i 'll need a departure from cambridge on thursday
find me a train to peterborough
i want to go from cambridge to peterborough i plan on leaving after 2:15 pm
i am going to be traveling on tuesday
i need a train from stevenage to cambridge
are there any trains departing on tuesday and arriving by 9:00
id like to take a train to cambridge on wednesday
my starting point is london kings cross
i need to get to kings cross departing from london
the user wants a travel that will arrive tuesday in cambridge by 17:00
the user needs a train for cambrigde that leaves sunday after 20:30 and passes by that street
is there a train that leaves closer to 20:30
sunday i need to go
yes can you please give me a ticket for 8 people
i want two tickets to cambridge please
does the train depart on friday
thank you for the help
train from ely to camebridge
i 'm looking to arrive by 14:15 leaving friday
the user is leaving stevenage to go to cambridge
yes book it for 8 and give me the reference number too
i would like to depart from stevenage on saturday
after 12:15 i want to leave
i would like to reserve a train arriving by 19:00 friday
i will be leaving from cambridge
destination is stansted airport
to combridge and should depart from london liver pool street
the user needs a train that will take him from cambridge to broxbourne
i 'll be riding the train on monday
can i reserve a seat for a train departing cambridge and arriving at bishops startford
i 'd like to leave on thursday after 20:15
hi i would like a train that leaves cambridge anytime after 9pm on thursday
i 'm searching for a saturday train to cambridge
of course i want on saturday please
by 13:45
i want a train leaving on wednesday and arriving 12:30
from cambirdge to london liverpool street
the user wants a train that leaves for leicester on tuesday
i am looking to depart after 11:15
looking to depart leicester after 11:15 and arrive at cambridge
i 'm looking for a train departing saturday from leicester
i intend to go to cambridge and and be there after midday
i am looking for a train from london kings cross and leaves on wednesday
on friday i need to leave on train from liverpool street london
arrival time should be 14:00 or earlier
user: i 'm looking for a way to get into a train that would take me from norwich to cambridge
it would be best to leave tuesday arriving by 13:30
the user wants a train on friday that leaves after 21:45
i shall be leaving from cambridge and my destination is stansted airport
i want to depart after 21:45
after 7 pm i will be departing london kingâ€™s cross
i need tickets for 6 people
i 'm searching for a train departing from london kings cross with an arrival time of 14:30
i will be arriving at cambridge
i will book for 7 sometime near 10 oclock
leaving from cambridge
i 'm available on a thursday so it 's a good day
17:45 arrival at the latest please
i want to catch a train from cambridge this sunday
i need to end up in leicester
i will want to leave after 10:15
i need a train which goes from bishops stortford to cambridge
i need for saturday
by 8:15 if it could be possible
i 'd like a train from london kings cross to cambridge
do the trains run on sunday
i want to find a train leaving from stevenage
on friday i need to get to cambridge
i will want to leave after 13:30
