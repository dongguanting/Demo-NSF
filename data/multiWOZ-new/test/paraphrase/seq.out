O O O O O O O O O O O B-day O O O B-arrive
O O O O O B-depart
O O O B-people
O O B-depart I-depart O B-leave
O O O O O O B-leave O B-depart
O O O O O O O O B-day
O O O O O O B-depart I-depart I-depart
O O B-leave O O
O O O O O O B-dest
O O O B-depart O B-day
O O O O O O B-arrive
O O O O B-people O
O O O O O B-dest O O O O O O O O O B-arrive
O O B-depart
O O O O B-dest O B-depart O B-day
O O O O O B-leave
O O O O O B-arrive O O O O O O O
O O O O O O O B-depart I-depart I-depart
O B-day O O O O O O O B-dest
O O O O O O B-dest O O O O B-arrive
O O O B-people O O
O O O O B-day O B-leave
B-arrive O
O O O O O O O O O B-day O B-arrive
O O O B-depart I-depart I-depart O O O B-dest
O B-arrive O O O O O O O O B-dest O
O O O O B-day
O O O O O O O B-leave O B-depart
O O B-dest I-dest O B-day
O O O O O O O B-day O B-depart
O O O O B-people O O O
O O O O O O B-dest I-dest I-dest O O B-leave
O O O O O O B-day
O O O O O O O O O B-depart O B-day
O O O O O O O O
O O O O B-depart
O O O O O O O B-dest I-dest I-dest
O O O O B-day
O B-people O O O
O O O O B-depart I-depart O O O
O O O O O O B-day
O O O O O O O O O O B-depart O B-day
O O O O B-dest I-dest O B-leave
O O O O O O B-depart O O O B-day
O O O O O O O O B-dest O B-arrive
O O O O B-day O B-dest I-dest
O O O O O O O O O B-depart
O O O O O B-dest O B-day
O O B-depart O B-day
O O O O O O O B-people O O
O O O O O O B-depart O B-dest O
O B-leave O O O O
O O O B-people O
O O O O O O O O B-depart O B-dest I-dest
O B-people O O
O O O O B-dest I-dest I-dest O O O O O O B-arrive
O O O O O O O B-dest
O O O O O O O O O O B-depart I-depart O B-leave O B-day
O O O O O O O B-depart
O O O O O O O O B-day O B-depart
O O O O O O O B-dest O O
O O O O O O B-depart
O O O O O O O B-dest I-dest O O
O O O O O O O B-dest
O O O O O O O O B-depart O
O O O O O O B-day
O O O O B-leave I-leave
O O O O O O O O O B-depart O B-dest
O O O O O O O O B-leave O O O O O O O
O O O O O B-dest I-dest O O O B-depart O
O O O O O B-dest
O O O O O B-dest O B-leave
O O O O O B-people O
O O O O O B-depart O
O O O O O O B-day
O O O O O O B-arrive
O O O B-people O O
O O O O O O B-day
O O O O O O B-leave
O O O B-people O
O O O O O B-depart O B-dest I-dest I-dest O B-day O B-arrive
O O O O B-day O O O B-arrive
O O B-depart O O O O
O O O O O O B-dest I-dest I-dest O B-day
O O O O B-depart O O O O O O B-arrive
O B-depart O O O O
O O O O O O B-day
O O O O B-day O O O B-depart
O O O O O O B-dest I-dest
O O O O O O O O B-depart O B-dest
O O O O O O B-dest I-dest I-dest O O
O O O O O O O O O B-dest
O B-day
O O O O O O O O O B-arrive
O O O O B-dest O B-depart
O O O O O O B-day
O O O O O O B-leave
O B-people O O
O O O O O B-depart O O O B-day
O O O O O O O B-leave O O O B-dest
O O O O O O O B-arrive
O O O O B-people
O B-day O O O O O O O O O O O B-arrive
O O O O O O B-depart O B-leave
O O O O B-depart I-depart O B-day O O O B-dest O B-arrive
O O O O O O O O O B-arrive
O O O O O O O B-dest I-dest I-dest O O O B-leave
O O O O O O B-depart O B-leave
O O O O O O O B-dest I-dest
O O B-day O O O O
O O O O B-leave I-leave O B-day
O O O O O B-dest I-dest
O O O O O B-arrive
O O B-day O O O
O O B-dest I-dest I-dest O O O O O B-day
O O O O O O O B-depart
O O O O O B-arrive O B-dest I-dest
O O O O O O O O O O O O B-day
O O O O O O O O B-dest I-dest O B-day O O O O B-arrive
O O O O O O O O O B-day
O O O O O O O B-people O O O O O O O O
O O O O O O B-depart
O O O O O O B-depart O O O B-dest I-dest
O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O
O O O O O B-depart O B-leave
O O O O O O O O O B-leave O O O B-dest
O O O O O O O B-day O
O O O O O O O B-day
O O O O B-dest O B-depart O B-leave O O O
O B-day O O O O O
O O O O O B-dest I-dest O O O B-leave
O O O O B-depart B-day
O O O O O B-depart
O O O O O B-dest
O O O O O B-arrive
O O O O B-day
O O O O O B-dest O B-arrive O
O O O O O O O B-dest
O O O O O B-depart O B-day
O O O O O B-leave
O O O B-day O O B-leave
O O O O O O B-day O O O O O
O O O O O O O B-leave
O O B-people O
O O O O O O O O B-depart O O O B-dest
O B-arrive O O O O O O
O O B-people O
O O O O O O O O O B-day O B-dest
O O O O O O O O B-depart
O O O O O O O O B-dest I-dest O
O O O O O B-day O
O O O O O O B-leave
O O O O O O O O B-people O O O O O B-day O O B-arrive O O B-dest O O B-depart
O O O O B-day O O O O B-arrive
O B-day O O O O O B-leave
O O O O O O B-dest
O O O O O B-dest B-day
O B-people O
O O O O O O O O O B-depart O O O B-dest O B-arrive
O O O O B-day O B-depart O
O O O O O O B-arrive
O O O O O O B-day
O O O O O O O O B-people O O O O
O O O O O O O O B-depart O B-dest
O O O B-dest I-dest
O O O O O O O B-depart
O O O O O B-dest O O O B-leave
O O O B-depart O O O O B-dest
O O O O O O O O B-dest O B-arrive
O O O O O O B-depart O B-day
O O O O O O O O B-dest
O O O O O O O O O B-dest
O O O O O O O O B-depart
O O O O O O B-dest O B-depart I-depart I-depart
O O O O O O O O O O O O O O O O 
O O B-people O O
O O O O O O O O O O O O B-dest I-dest I-dest
O O O O O O B-day
O O O O O O O B-depart O B-dest O O B-day
O O O B-leave I-leave O O O
O O O O O B-depart O B-dest
O O O O B-day
O O O O O O O B-dest
O O O O O B-leave O O O O B-people O
O O O O O O B-day B-leave I-leave
O O O O B-dest
O O O O O O B-depart I-depart I-depart O
O O O O O O B-leave
O O O O O B-depart O O O O O O O O O O B-dest I-dest O O
O O O O O O O O O B-depart
O O O O O B-arrive B-day
O O O O O O O B-dest O
O O O O O O B-dest I-dest I-dest O B-day
O O O O O O O B-arrive
O O O O O O O O B-depart O B-dest O O O B-arrive
O O O O O O B-day O O O O
O O O O O B-depart O B-dest
O O O O O O O O O O O O O O O O O
O O O O O O O O
O O O B-depart I-depart O O B-dest
B-day O O O O B-leave
O O O O O O O O O O O B-day
B-day O O O O
O O O O O B-arrive O O O O
O O O O B-depart O B-dest
O O O O O O O O O O O B-dest
O O O O O B-depart O O O O O O B-day
O O O B-depart O O O O O O B-leave
O O O O O B-depart I-depart O B-day
O O O O O B-day
O O B-day O O B-depart I-depart O B-dest
O B-day O O O O O O B-dest I-dest I-dest
O O O O B-depart O B-leave
O O B-dest
O O O B-depart O B-dest
O O O O O B-people O O O O O O
O O O O O B-leave
O O O O O B-depart I-depart O B-dest O B-day
O O O O O O O O O O O O O O O O O O O O O B-people O
O O O O O B-leave O B-dest I-dest I-dest
O O O O O O O O B-dest O O B-arrive
O O O O O B-depart O B-dest
O O O O O O O B-day O O O B-depart
O O O B-dest
O O O O O O O O B-arrive
O O O B-people O
O O O O O O O B-dest O B-depart O B-day
O O O O O O B-leave
O O B-depart
O O O O O B-leave
O O O O O O O O O O O O O B-depart
O O O O O O O O O O O B-dest O B-depart I-depart I-depart O B-day
O O O O O O O B-dest
O O O O O O O O O O O O O B-depart I-depart
B-day O O O O O O O O
O O O O O O B-leave
O O O O O O B-depart
O O O O O O B-depart O B-day
O O O O O B-dest
O O O O O B-depart O O O O O O O B-leave I-leave
O O O O O O O B-day
O O O O O B-depart O B-dest
O O O O O O B-day O O O B-arrive
O O O O O O O B-dest O B-day
O O O O B-depart I-depart I-depart
O O O O O B-depart I-depart O O O
O O O O O O O O B-day O B-dest O B-arrive
O O O O O O O O O B-day O B-leave O O O O O
O O O O O O O O B-leave
B-day O O O O
O O O O O O O O O B-people O
O O B-people O O O O
O O O O O B-day
B-dest O O O O
O O B-depart O B-dest
O O O O O O B-arrive O B-day
O O O O B-depart O O O B-dest
O O O O B-people O O O O O O O
O O O O O O B-depart O B-day
O B-leave O O O O
O O O O O O O O O B-arrive B-day
O O O O O B-depart
O O B-dest I-dest
O B-dest O O O O B-depart I-depart I-depart I-depart
O O O O O O O O O O B-depart O B-dest
O O O O O O O B-day
O O O O O O O O O B-depart O O O B-dest I-dest
O O O O O O B-day O B-leave
O O O O O O O O B-depart O O B-leave O B-day
O O O O O B-day O O B-dest
O O O O O B-day O
O B-arrive
O O O O O O B-day O O B-arrive
O B-depart O B-dest I-dest I-dest
O O O O O O O O B-depart O B-day
O O O O O O B-leave
O O O O O O O O O B-dest
O O O O O O O B-day O O
O O O O O B-dest O O O O O B-day
O O O O O O O B-depart I-depart I-depart O O O B-day
O B-day O O O O O O O B-depart I-depart I-depart
O O O O B-arrive O O
O O O O O O O O O O O O O O O O O B-depart O B-dest
O O O O O O B-day O O B-arrive
O O O O O O B-day O O O B-leave
O O O O O B-depart O O O O B-dest I-dest
O O O O O B-leave
O B-leave I-leave O O O O B-depart I-depart I-depart
O O O O B-people O
O O O O O O O O B-depart I-depart I-depart O O O O O B-arrive
O O O O O B-dest
O O O O B-people O O B-leave O
O O B-depart
O O O O O B-day O O O O O O
B-arrive O O O O O
O O O O O O O B-depart O B-day
O O O O O O B-dest
O O O O O O B-leave
O O O O O O O B-depart I-depart O B-dest
O O O B-day
O B-arrive O O O O O
O O O O O O B-depart I-depart I-depart O B-dest
O O O O O B-day
O O O O O O O O B-depart
O B-day O O O O O B-dest
O O O O O O B-leave
